frog/imports/client/Wiki/index.js

```diff
@@ -182,12 +194,76 @@ class WikiComp extends React.Component<WikiCompPropsT, WikiCompStateT> {
       );
     }
 
+    // Show locked modal if the wiki is locked
+    if (
+      this.wikiDoc.data.settings?.locked &&
+      !this.wikiDoc.data.owners.find(x => x === Meteor.userId())
```

Comment:
This call should be moved to its own function or saved to a variable as we are repeating ourselves

Reply:
We are checking the existence of the user in 3 different lists, `owners`, `editors` & `users`